homeassistant/components/lametric/manifest.json

```diff
@@ -4,7 +4,7 @@
   "documentation": "https://www.home-assistant.io/integrations/lametric",
   "requirements": ["demetriek==0.2.2"],
   "codeowners": ["@robbiet480", "@frenck"],
-  "iot_class": "local_push",
+  "iot_class": "local_polling",
```

Comment:
Shouldn't we keep it at the best device class, or is all local push gone?

Reply:
It previously only had a push notification out of HA.
Now it polls data into HA, so want to make it fair.